src/CabanaPD_Comm.hpp

```diff
@@ -398,13 +398,14 @@ class Comm<ParticleType, Pair, MaterialType, TemperatureDependent>
     void gatherTemperature() { gather_temp->apply(); }
 };
 
-template <class ParticleType, class ModelType, class ThermalType>
-class Comm<ParticleType, ModelType, MultiMaterial, ThermalType>
-    : public Comm<ParticleType, ModelType, SingleMaterial, ThermalType>
+template <class ParticleType, class ModelType>
+class Comm<ParticleType, ModelType, MultiMaterial, TemperatureIndependent>
+    : public Comm<ParticleType, ModelType, SingleMaterial,
+                  TemperatureIndependent>
```

Comment:
I restricted this since otherwise there are multiple specializations that could fit a Thermal simulation with multiple materials. @streeve do we need the methods in this `Comm` implementation for `MultiMaterial` to be executed? If so, we need to think about how to redesign the set of specialziations to make it non-ambiguous

Reply:
Makes sense; that's my mistake